File: ruby/astronauts.rb
require 'net/http'
require 'json'

response = JSON.parse Net::HTTP.get URI "http://api.open-notify.org/astros.json"
printf "%25s | %10s\n", "NAME", "CRAFT"
response["people"].each do |astronaut|
  printf "%25s | %10s\n", astronaut["name"], astronaut["craft"]
end

File: javascript/typescript/astronauts.ts
const astronauts = await fetch("http://api.open-notify.org/astros.json").then(response => response.json());
[{ name: "NAME", craft: "CRAFT" }, ...astronauts.people].forEach(({ name, craft }) => {
  console.log("%s | %s", name.padStart(25, " "), craft.padStart(10, " "));
});
